Introduce a modern, JSX-like syntax for TwigComponents (#2662)
symfony/ux: src/TwigComponent/CHANGELOG.md

# CHANGELOG

> ## 2.21.0

Reviewer: ```suggestion
## 2.21.0
```

`2.24` i think ?  (feel free to create a new section here)

> -  Introduce an experimental JSX-like syntax for TwigComponents, making `<twig:` prefix optional

Reviewer 2: ```suggestion
-  Introduce an experimental JSX-like syntax for TwigComponents, making `twig:` prefix optional
```

> -  Introduce an experimental JSX-like syntax for TwigComponents, making `<twig:` prefix optional

Reviewer: ```suggestion
-  Add experimental support for prefix-less tags in HTML Syntax (`<FooBar />` instead of `<twig:FooBar />`)
```
